__test__/features/user/userAPI.test.tsx

```diff
@@ -20,3 +23,54 @@ describe("getOpenPaasUserId", () => {
     expect(result).toEqual(mockUser);
   });
 });
+
+describe("updateUserConfigurations", () => {
+  it("should PUT configurations with language update", async () => {
+    const mockResponse = { status: 204 };
+    (api.put as jest.Mock).mockResolvedValue(mockResponse);
+
+    await updateUserConfigurations({ language: "vi" });
+
+    expect(api.put).toHaveBeenCalledWith("api/configurations?scope=user", {
+      json: [
+        {
+          name: "core",
+          configurations: [{ name: "language", value: "vi" }],
+        },
+      ],
+    });
+  });
+
+  it("should PUT configurations with multiple updates", async () => {
```

Comment:
With PATCH we do not need to care and we do not need to have the merging logic in the front.